src/Traits/PivotEventTrait.php

```diff
@@ -23,6 +24,16 @@ public function getObservableEvents()
         );
     }
 
+    public static function pivotSyncing($callback, $priority = 0)
+    {
+        static::registerModelEvent('pivotSyncing', $callback, $priority);
+    }
+
+    public static function pivotSynced($callback, $priority = 0)
+    {
+        static::registerModelEvent('pivotSynced', $callback, $priority);
+    }
```

Comment:
Nice, these sync events are good additions.

Reply:
Thanks, glad you appreciate it 😄